### PR TITLE
chore(docs/examples): Adding GraphQL information

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds the graphql docs and recipes

## Changes

- Adds the docs for graphql trigger
- Adds a recipe for Graphql

## Fixes

- https://github.com/kubeshop/tracetest-cloud/issues/854 (partially)

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by danielbdias — APPROVED** [review]

File: docs/docs/examples-tutorials/recipes/running-tests-with-tracetest-graphql-pokeshop.mdx [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+[GraphQL](https://graphql.org/) is a query language for APIs and a runtime for fulfilling those queries with your existing data. GraphQL provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.
+
+## Why is this important?
+
+The Tracetest GraphQL trigger enables you to unleash the power of the trace-based testing to easily capture a full distributed trace from your OpenTelemetry instrumented GraphQL back-end system.
+
+By creating a Tracetest GraphQL test, you will be able to create trace-based assertions to be applied across the entire flow like any other Tracetest test. Not only that but it allows you to mix and match it with your existing Monitors, Test Suites and CI/CD validations.
+
+Another big benefit of using traces as test specs is that you can:
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-tests-with-tracetest-graphql-pokeshop.mdx:36]:
```suggestion
Other impactful benefits of using traces as test specs are:
```

File: docs/docs/examples-tutorials/recipes/running-tests-with-tracetest-graphql-pokeshop.mdx [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+
+- Get faster MTTR for failing performance tests
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-tests-with-tracetest-graphql-pokeshop.mdx:38]:
```suggestion
- Faster MTTR for failing performance tests.
```

File: docs/docs/examples-tutorials/recipes/running-tests-with-tracetest-graphql-pokeshop.mdx [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+- Assert against the Mutiple Queries and Mutations at once from a single test execution
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-tests-with-tracetest-graphql-pokeshop.mdx:39]:
```suggestion
- Assert against the Mutiple Queries and Mutations at once from a single test execution.
```

File: docs/docs/examples-tutorials/recipes/running-tests-with-tracetest-graphql-pokeshop.mdx [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+- Validate functionality of other parts of your system that may be broken, even when the initial request is passing
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-tests-with-tracetest-graphql-pokeshop.mdx:40]:
```suggestion
- Validate functionality of other parts of your system that may be broken, even when the initial request is passing.
```

File: examples/tracetest-jaeger-graphql-pokeshop/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Tracetest + Amazon X-Ray (using Tracetest awsxray integration) + Pokeshop
+
+> [Read the detailed recipe for setting up Tracetest + Amazon X-Ray (using Tracetest awsxray integration) + Pokeshop in our documentation.](https://docs.tracetest.io/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-pokeshop)
+
+This repository objective is to show how you can configure your Tracetest instance to connect to AWS X-Ray and use it as its tracing backend.
+
+## Steps
+
+1. [Install the tracetest CLI](https://docs.tracetest.io/installing/)
```
Comment (jfermi) [inline examples/tracetest-jaeger-graphql-pokeshop/README.md:9]:
```suggestion
1. [Install the tracetest CLI](https://docs.tracetest.io/installing/).
```

File: examples/tracetest-jaeger-graphql-pokeshop/README.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+2. Run `tracetest configure --server-url http://localhost:11633` on a terminal
```
Comment (jfermi) [inline examples/tracetest-jaeger-graphql-pokeshop/README.md:10]:
```suggestion
2. Run `tracetest configure --server-url http://localhost:11633` on a terminal.
```

File: examples/tracetest-jaeger-graphql-pokeshop/README.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+3. Update the `.env` file adding a valid set of AWS credentials
```
Comment (jfermi) [inline examples/tracetest-jaeger-graphql-pokeshop/README.md:11]:
```suggestion
3. Update the `.env` file adding a valid set of AWS credentials.
```

File: examples/tracetest-jaeger-graphql-pokeshop/README.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+4. Update the `tracetest.provision.yaml` file adding a valid set of AWS credentials
```
Comment (jfermi) [inline examples/tracetest-jaeger-graphql-pokeshop/README.md:12]:
```suggestion
4. Update the `tracetest.provision.yaml` file adding a valid set of AWS credentials.
```

File: examples/tracetest-jaeger-graphql-pokeshop/README.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+5. Run the project by using docker-compose: `docker-compose up -d` (Linux) or `docker compose up -d` (Mac)
```
Comment (jfermi) [inline examples/tracetest-jaeger-graphql-pokeshop/README.md:13]:
```suggestion
5. Run the project by using docker-compose: `docker-compose up -d` (Linux) or `docker compose up -d` (Mac).
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Pull request merged**